kubernetes-extension-examples/examples/knative-deployment/knative_deployment.out

```diff
@@ -0,0 +1,37 @@
+# Build the ballerina program
+$ ballerina build knative_deployment.bal
+Compiling source
+	knative_deployment.bal
+
+Generating executables
+	knative_deployment.jar
+
+Generating Knative artifacts...
+
+	@knative:Service 			 - complete 1/1
+	@knative:Docker 			 - complete 3/3
+
+	Run the following command to deploy the Knative artifacts:
```

Comment:
```suggestion
	Execute the below command to deploy the Knative artifacts:
```

Reply:
These are outputs of the build command with annotations.